diff --git a/arch/arm/dts/tegra124-nyan-big-u-boot.dtsi b/arch/arm/dts/tegra124-nyan-big-u-boot.dtsi
index 3ff2f67c1a..ddfeba806c 100644
--- a/arch/arm/dts/tegra124-nyan-big-u-boot.dtsi
+++ b/arch/arm/dts/tegra124-nyan-big-u-boot.dtsi
@@ -15,7 +15,7 @@
 	};
 
 	spi@7000d400 {
-		spi-deactivate-delay = <200>;
+		spi-deactivate-delay = <500>;
 		spi-max-frequency = <3000000>;
 
 		cros_ec: cros-ec@0 {
